govuk_frontend_jinja/templates/components/fieldset/macro.html

```diff
@@ -1,21 +1,27 @@
 {% macro govukFieldset(params) %}
-  <fieldset class="govuk-fieldset
+<fieldset class="govuk-fieldset
   {%- if params.classes %} {{ params.classes }}{% endif %}"
   {%- if params.role %} role="{{ params.role }}"{% endif %}
   {%- if params.describedBy %} aria-describedby="{{ params.describedBy }}"{% endif %}
   {%- for attribute, value in (params.attributes.items() if params.attributes else {}.items()) %} {{ attribute }}="{{ value }}"{% endfor %}>
-  {% if params.legend.html or params.legend.text %}
-  <legend class="govuk-fieldset__legend {%- if params.legend.classes %} {{ params.legend.classes }}{% endif %}">
-  {% if params.legend.isPageHeading %}
-    <h1 class="govuk-fieldset__heading">
-      {{ params.legend.html | safe if params.legend.html else params.legend.text }}
-    </h1>
-  {% else %}
-    {{ params.legend.html | safe if params.legend.html else params.legend.text }}
+  {% if params.legend %}
+    {% if params.legend.html or params.legend.text %}
+    <legend class="govuk-fieldset__legend {%- if params.legend.classes %} {{ params.legend.classes }}{% endif %}">
+      {% if params.legend.isPageHeading %}
+        <h1 class="govuk-fieldset__heading">
+          {{ params.legend.html | safe if params.legend.html else params.legend.text }}
+        </h1>
+      {% else %}
+        {{ params.legend.html | safe if params.legend.html else params.legend.text }}
+      {% endif %}
+    </legend>
+    {% endif %}
   {% endif %}
-  </legend>
+  {% if caller %} {#- if statement allows usage of `call` to be optional -#}
+    {{ caller() }}
+  {% elif params.html %}
+    {{ params.html | safe }}
```

Comment:
definitely safe?

Reply:
This has come from upstream in the govuk-frontend Nunjucks - I wonder whether it's actually worth a post on the govuk-frontend GitHub repo querying the usage. I tend towards not using this where possible and if you're passing html it's up to you to pass it through the MarkupSafe api first (meaning Jinja won't then autoescape it).

You _could_ consider deviating from it in this library, but it's probably better to first explore the issue at the gov end and bottom it out there, even if it's just a quick chat on the X-Gov slack.

Reply:
Although I guess you could argue that by passing it into the `html` param instead of `text`, you are explicitly getting the developer to pass html and they shouldn't expect it to be escaped. I.e. if you don't have safe content, use `text`. If you have to pass html use `html` and it won't be escaped.

Reply:
Would rather remain consistent with the upstream original for this port, so that expected behaviour is the same. I think the explicit use of `html` rather than `text` is the key point.